torch2trt/converters/cat.py

```diff
@@ -16,7 +16,7 @@ def convert_cat(ctx):
     trt_inputs = broadcast_trt_tensors(ctx.network, trt_inputs, len(output.shape) - 1)
```

Comment:
Just pattern matching, but do we need to remove the `- 1` here? ie. 
```
    trt_inputs = broadcast_trt_tensors(ctx.network, trt_inputs, len(output.shape))
```
I assume the dim reduction here was also due to the previous removal of the batch dim.